src/main/java/org/apache/maven/xinclude/stax/XIncludeStreamReader.java

```diff
@@ -70,9 +71,19 @@
 @SuppressWarnings("checkstyle:MissingSwitchDefault")
 class XIncludeStreamReader extends StreamReaderDelegate {
 
-    private static final String XINCLUDE_NAMESPACE = "http://www.w3.org/2001/XInclude";
-    private static final String XINCLUDE_INCLUDE = "include";
-    private static final String XINCLUDE_FALLBACK = "fallback";
+    public static final String XINCLUDE_NAMESPACE = "http://www.w3.org/2001/XInclude";
```

Comment:
Maybe this one. The rest are not helpful. Named constants clarify numbers like 0.75 but they are not better than literal strings and usually worse.  For instance  "include" is much simpler, clearer, and more obvious than XINCLUDE_INCLUDE_ELEMENT. 

Also note that "include" is not an element but rather an element name.